src/OnlineSales/Program.cs

```diff
@@ -131,6 +134,58 @@ public static void Main(string[] args)
         app.Run();
     }
 
+    private static void ConfigureImportSizeLimit(WebApplicationBuilder builder)
+    {
+        string fileSize;
+        string measurement;
+        int size;
+        // With default size
+        long sizeInByte = 30 * 1024 * 1024;
+
+        var maxImportSize = builder.Configuration.GetValue<string>("ApiSettings:MaxImportSize");
+
+        if (string.IsNullOrEmpty(maxImportSize))
+        {
+            throw new MissingConfigurationException("Import file size is mandatory.");
+        }
+
+        measurement = maxImportSize![^2..];
```

Comment:
The code below was copied and pasted 3 times. 2 times by Sandaruwan and now you copied it the 3rd time. It breaks my heart...